gnosis/safe-react
# style: redesign stepper

## What it solves
Resolves #3375

## How this PR fixes it
The transaction status stepper has been updated to match the [Figma design](https://www.figma.com/file/47wkjxMZkyEeCRbCrLJBJa/safe%232995-Tx-details?node-id=616%3A20938).

Confirmations are hidden by default when either:
- The threshold has been reached.
- There are >3 confirmations.

## How to test it
The stepper should match the Figma design linked above, tested as so:

1. Create a transaction on a 1/? Safe and execute it immediately. Observe that:
    - The creation logo is an outlined plus.
    - Confirmations (with address) are filled, with the possibility to hide/show all.
    - 'Executing' is shown (with no threshold description) in gray, eventually changing to 'Executed' in green.
2. Create a transaction on a 2/? Safe. Observe (as well as the above) that:
    - The confirmations title is in orange and changes to green when the threshold is reached.
    - 'Execution' is shown (with a description about threshold requirement), hiding the description when the threshold is reached.
3. Create a transaction on a 4/? Safe. Observe (as well as the above) that:
    - From the third confirmation, the confirmations are hidden by default.
4. Create an on-chain rejection. Observe (as well as the above) that:
    - Creation logo is an outlined cross.
5. View historical transactions on a 1/? or 2/? Safe. Observe (as well as the above) that:
    - Creation, confirmations (hidden by default) and executed are in green. Each showing the address off the related party.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/151954692-1927d3a6-7bb0-4299-99c2-336944a02837.png)

![image](https://user-images.githubusercontent.com/20442784/151954730-e3a9d876-0156-4120-bce9-5384fec7b4c7.png)

![image](https://user-images.githubusercontent.com/20442784/151954760-2dd17b2f-c4a2-4edd-b8ac-804bb100ae86.png)
